File: Python/1 - Iniciante/1061.py
# coding: utf-8
# Funcional no URI

d1 = input().split()
h1 = input().split()
d2 = input().split()
h2 = input().split()

dc = [int(d1[1]), int(h1[0]), int(h1[2]), int(h1[4])]
df = [int(d2[1]), int(h2[0]), int(h2[2]), int(h2[4])]

min_seg = 60
hor_seg = min_seg * 60
dia_seg = hor_seg * 24

i = dc[3] + dc[2] * min_seg + dc[1] * hor_seg + dc[0] * dia_seg
f = df[3] + df[2] * min_seg + df[1] * hor_seg + df[0] * dia_seg

if i < f:
    T = f - i
    d = int(T/dia_seg)
    T = T % dia_seg
    H = int(T/hor_seg)
    T = T % hor_seg
    M = int(T/min_seg)
    T = T % min_seg
    S = T

print('{} dia(s)\n{} hora(s)\n{} minuto(s)\n{} segundo(s)'.format(d,H,M,S))

File: go/1061.go
// Funcional no beecrowd
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {

	// Inicializador do Scanner
	in := bufio.NewScanner(os.Stdin)

	// Leitura de linhas
	in.Scan()
	d1 := in.Text()
	in.Scan()
	h1 := in.Text()
	in.Scan()
	d2 := in.Text()
	in.Scan()
	h2 := in.Text()

	// Separação das strings
	di := strings.Split(d1, " ")
	hi := strings.Split(h1, ":")
	df := strings.Split(d2, " ")
	hf := strings.Split(h2, ":")

	// Conversão de valores de string para inteiro
	var horaI []int
	var horaF []int

	diaI, _ := strconv.Atoi(di[1])
	diaF, _ := strconv.Atoi(df[1])

	for _, i := range hi {
		valor, _ := strconv.Atoi(strings.TrimSpace(i))
		horaI = append(horaI, valor)
	}

	for _, i := range hf {
		valor, _ := strconv.Atoi(strings.TrimSpace(i))
		horaF = append(horaF, valor)
	}

	// Valores iniciais de minuto hora e dia em segundos
	min := 60
	hor := 60 * 60
	dia := 60 * 60 * 24

	// Soma dos valores e transformação em segundos
	i := horaI[2] + horaI[1]*min + horaI[0]*hor + diaI*dia
	f := horaF[2] + horaF[1]*min + horaF[0]*hor + diaF*dia

	// Cálculo de valores para respectivos valores de Dia, Minuto, Hora e Segundo do restante
	if i < f {
		T := f - i
		d := int(T / dia)
		T = T % dia
		H := int(T / hor)
		T = T % hor
		M := int(T / min)
		T = T % min
		S := T
		fmt.Printf("%d dia(s)\n%d hora(s)\n%d minuto(s)\n%d segundo(s)\n", d, H, M, S)
	}
}
